sql/b_trans.sql

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Fixed. Originally we thought this would stand for 'block transfer'. Renamed to ver_transfers.